pkg/dir/dir.go

```diff
@@ -20,6 +20,10 @@ type Prefix struct {
 
 	Mode int
 
+	ChOwn
+}
+
+type ChOwn struct {
```

Comment:
Uh. You're not _seriously_ asking me to approve a PR with a struct `ChOwn`, are you? 

"It's not just awful, it's _god_awful." 😂

Reply:
(Maybe `Ownership` or `FileOwner` or some such?)